Classes/Controller/BuilderModuleController.php

```diff
@@ -262,10 +278,22 @@ protected function rpcActionSave()
         }
 
         $extensionDirectory = $extension->getExtensionDir();
+        $publicExtensionDirectory = Environment::getExtensionsPath() . '/' . $extension->getExtensionKey();
+        $usesComposerPath = $this->extensionService->isComposerStoragePath($extensionDirectory);
+        $extensionExistedBefore = is_dir($extensionDirectory);
 
-        if (!is_dir($extensionDirectory)) {
+        if (!$extensionExistedBefore) {
             GeneralUtility::mkdir($extensionDirectory);
-        } else {
+            $extensionExistedBefore = false;
```

Comment:
Why setting this to `false` again if the condition above is already ensuring this is `false`?